src/main/java/by/itech/lab/supplier/config/WebConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package by.itech.lab.supplier.config;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.List;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"by.itech.lab.supplier"})
+public class WebConfiguration implements WebMvcConfigurer {
+
+    private final long MAX_AGE_SECS = 3600;
+
+    @Override
+    public void configureMessageConverters(
+            List<HttpMessageConverter<?>> converters) {
+        converters.add(new MappingJackson2HttpMessageConverter(
+                new Jackson2ObjectMapperBuilder().serializationInclusion(JsonInclude.Include.NON_NULL)
```

Comment:
What about configuring of inclusion with props, not with replacing of original converter?

Reply:
for example
spring.jackson.serialization-inclusion=non_null